src/Networking/real_world_interface.cpp

```diff
@@ -75,6 +75,8 @@ constexpr double EFF_WHEEL_BASE = 1.40;
 
 constexpr double WHEEL_RADIUS = 0.15;			 // Eyeballed
 constexpr double PWM_FOR_1RAD_PER_SEC = 5000; // Eyeballed
+// This is a bit on the conservative side, but we heard an ominous popping sound at 20000.
```

Comment:
Does it still make the ominous popping sound?

Reply:
Oh right. No, it does not. I left the comment as a warning to be careful about trying yet higher PWMs, but yes I should clean that up at some point.